src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.xml

```diff
@@ -58,8 +57,21 @@
                 <include refid="cnaAlterationFilter"/>
             </if>
         </where>
-        GROUP BY hugo_gene_symbol, alteration, cytoband
-        ORDER BY totalCount DESC;
+        GROUP BY hugo_gene_symbol, alteration, cytoband;
+    </select>
+
+    <select id="getStructuralVariantGenes" resultType="org.cbioportal.model.AlterationCountByGene">
+        SELECT
+        hugo_gene_symbol as hugoGeneSymbol,
+        1 as entrezGeneId,
```

Comment:
there's really a field "1"?

Reply:
I just haven't added back the entrezID not really needed. Even on the frontend. Just gotta update the table

Reply:
this is probably some SQL convetnion i don't know about.